network/omaha/protocol.go

```diff
@@ -48,6 +48,8 @@ type Request struct {
 
 func NewRequest() *Request {
 	return &Request{
+		// silly hack, xml's Encoder doesn't want to set this.
```

Comment:
the xml encoder does not set this but the decoder does, leading to a diff in the structs when i compare them in server_test.go.

see https://travis-ci.org/coreos/mantle/jobs/98414566 for the diff output